cmd/prometheus/main.go

```diff
@@ -262,6 +268,9 @@ func main() {
 	a.Flag("web.listen-address", "Address to listen on for UI, API, and telemetry.").
 		Default("0.0.0.0:9090").StringVar(&cfg.web.ListenAddress)
 
+	a.Flag("gomemlimit.ratio", "The ratio of detected memomry limit (container/system) used to set the GOMEMLIMIT").
```

Comment:
“container/system” doesn’t resonate with me. 
Could you explain a bit what is meant?

I expected some note about needing space for memory-mapped IO.

Reply:
It first checks to see if it's in a cgroup and sets the limit if it finds a cgroup memory limit. If no cgroup limit is found it checks to total system memory and sets the limit based on that.

I can add some more of these details in the feature flag documentation page.

Reply:
Ok but I’m asking on the “ratio” line.